modules/ROOT/pages/anypoint-platform-cli-commands.adoc

```diff
@@ -1836,25 +1967,30 @@ This command does not take any options, except for the default ones: `--help`, `
 > runtime-mgr serverGroup create [options] <name> [serverIds...]
 ----
 
-This command creates a server group with the name passed in `name` using the server Id(s) passed as argument(s) thereafter. +
-This command does not take any options, except for the default ones: `--help`, `-f`/`--fields` and `-o`/`--output`.
+This command creates a server group with the name passed in `name` using the server Id(s) passed as argument(s) thereafter.
```

Comment:
"using all IDs"? do we allow constructions like "argument(s)"?

Reply:
"arguments"

Reply:
Ditto